src/test/java/net/snowflake/client/RunningOnTravisCI.java

```diff
@@ -6,20 +6,15 @@
 /**
  * Run tests on CI
  */
-public class RunningOnTravisCI implements
-                               ConditionalIgnoreRule.IgnoreCondition
+public class RunningOnTravisCI implements ConditionalIgnoreRule.IgnoreCondition
```

Comment:
nit: this name could be changed from RunningOnTravis to RunningOnGithubActions

Reply:
updated.